functional-tests/src/test/resources/eap7-testrunner.json

```diff
@@ -69,7 +69,7 @@
       "initContainers": [
          {
             "name": "pem-to-truststore",
-            "image": "${DOCKER_REGISTRY_REDHAT}redhat-sso-7/sso71-openshift:1.1-16",
+            "image": "${image}",
```

Comment:
Shouldn't this be `${IMAGE}` with capital letters?

Reply:
I think it's using this: https://github.com/jboss-container-images/datagrid-7-image/blob/datagrid-services-dev/functional-tests/src/test/resources/arquillian.xml#L34 which is lower case.

Reply:
ooh right! I'm blind!